google/conf.py

```diff
@@ -21,6 +21,8 @@
         "../gce/create-customised-docker-container/",
     "google-how-to/upgrade-from-focal-to-jammy":
         "../gce/upgrade-from-focal-to-jammy/",
+    "google-how-to/check-cve-on-instance":
+        "../gce/check-cve-on-instance/",
```

Comment:
This redirection is not needed since we don't have an existing page on this topic in the google-how-to folder. These redirections were added when we introduced the sub-folders (contributions, gce and gke).

Reply:
Removed this redirection.